common/src/test/java/com/microsoft/identity/common/internal/providers/oauth2/StateGeneratorTest.java

```diff
@@ -0,0 +1,27 @@
+package com.microsoft.identity.common.internal.providers.oauth2;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class StateGeneratorTest {
+
+    private static final int TASK_ID = 19;
+    private static final String STATE_EXAMPLE_1 = String.format("%s:%s", TASK_ID, "SOMEGUID-SOMEGUID");
+
+    @Test
+    public void test_stateGeneratorGenerateMethod(){
+        StateGenerator generator = new AndroidTaskStateGenerator(TASK_ID);
+        String state = generator.generate();
+        String expected = String.valueOf(TASK_ID);
+        Assert.assertEquals(expected, state.split(":")[0]);
+    }
+
+    @Test
+    public void test_stateGeneratorParseMethod(){
```

Comment:
What is the parse method?

Reply:
I renamed this, but forgot to rename the test method.  Good catch.